packages/context/src/__tests__/acceptance/interceptor.acceptance.ts

```diff
@@ -187,7 +187,7 @@ describe('Interceptor', () => {
     }
 
     // No listeners yet
-    expect(ctx.listenerCount('bind')).to.eql(0);
+    expect(ctx.listenerCount('bind')).to.eql(1);
```

Comment:
Why is the listener count 1 considered as "no listeners yet"? Please update the code comment to explain.